.github/actions/install-grafana/action.yaml

```diff
@@ -0,0 +1,26 @@
+name: InstallGrafana
```

Comment:
Why are we installing Grafana in an ephemeral way?

Reply:
This was so that we could enable pyroscope when we add it in. I can remove the action itself from the workflow, but would want to keep the action in just for ease of use in the future.

Reply:
Got it, linking the docs for context: https://grafana.com/docs/pyroscope/latest/get-started/.